packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/mappers/ReactTextMapper.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay.mappers
+
+import android.widget.TextView
+import androidx.annotation.VisibleForTesting
+import com.datadog.android.sessionreplay.internal.AsyncJobStatusCallback
+import com.datadog.android.sessionreplay.internal.recorder.MappingContext
+import com.datadog.android.sessionreplay.internal.recorder.mapper.TextViewMapper
+import com.datadog.android.sessionreplay.model.MobileSegment
+import com.datadog.reactnative.sessionreplay.ReactTextShadowNodeUtils
+import com.datadog.reactnative.sessionreplay.ReactTextPropertiesResolver
+import com.datadog.reactnative.sessionreplay.extensions.resolveShapeAndBorder
+import com.datadog.reactnative.sessionreplay.getReactBackgroundFromDrawable
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.views.view.ReactViewBackgroundDrawable
+
+internal class ReactTextMapper(): TextViewMapper() {
+    private lateinit var reactTextPropertiesResolver: ReactTextPropertiesResolver
+
+    internal constructor(
+        reactContext: ReactContext
+    ): this() {
+        val reactTextShadowNodeUtils = ReactTextShadowNodeUtils(reactContext)
+        this.reactTextPropertiesResolver = ReactTextPropertiesResolver(reactTextShadowNodeUtils)
```

Comment:
My personal opinion, but I don't really like this pattern of creating a helper in the mapper only to pass it to another helper. It could be created in the helper directly.
Not a big deal though, if it enables testing we can leave it here.